packages/tiler/src/raster.ts

```diff
@@ -39,6 +39,8 @@ export enum ImageFormat {
     WEBP = 'webp',
 }
 
+export const ImageFormatDefaultLast = ['PNG', 'WEBP', 'JPEG'];
```

Comment:
if you made this a `ImageFormat[]` you would not need to do the casting to keyof typeof

`export const ImageFormatOrder = [ ImageFormat.PNG, ImageFormat.WEBP, ImageFormat.JPEG]`